plugin-infra/go-plugin-domain/src/main/java/com/thoughtworks/go/plugin/domain/analytics/AnalyticsData.java

```diff
@@ -43,7 +45,7 @@ public String getFullViewPath() {
         if (StringUtils.isBlank(assetRoot)) {
             return viewPath;
         }
-        return URI.create(assetRoot + "/" + viewPath).normalize().toString();
+        return URI.create(assetRoot + File.separator + viewPath).normalize().toString();
```

Comment:
Won't this use the wrong separator on windows?

Reply:
@ketan -- File.separator will give the right path separator based on the OS. References: https://docs.oracle.com/javase/7/docs/api/java/io/File.html#separator

But the problem is with `URI.create`.  Its failing if we provide `\\`. As the URI cannot contain backward slash.

Instead of URI we can use `Paths.get`.

\cc: @maheshp